spec/features/dashboard/index_spec.rb

```diff
@@ -68,6 +68,54 @@
           end
         end
 
+        describe 'friendship sad path' do
+          describe 'when I fill in the add friend form with an email that does not correspond to a user' do
```

Comment:
I appreciate the easy-to-understand describe blocks. Good stuff in this test overall.